api/types/updatable_project_fields.go

```diff
@@ -54,5 +75,29 @@ func (i *UpdatableProjectFields) Validate() error {
 			}
 		}
 	}
+
+	v := validator.New()
```

Comment:
`Validate` is designed to be thread-safe and used as a singleton. So I think creating an instance every time Validate is called is not fit the design intent.

https://github.com/go-playground/validator/blob/9e2ea4038020b5c7e3802a21cfa4e3afcfdcd276/validator_instance.go#L94-L99

Reply:
Thanks for review.
Now I make new file `validator.go` and use `Validate` as a package global variable.
- https://github.com/yorkie-team/yorkie/pull/345/files#diff-ad4ef0932407ad66ec522d5674a6995b785ac92d716d6c9ecd146ff870b181e5:~:text=var%20(-,defaultValidator,-%3D%20initValidator()

Please check it and let me know if you have anything in mind.